sum.js

```diff
@@ -0,0 +1,4 @@
+function sum(a, b) {
```

Comment:
I'm guessing this file is not part of this project functionality and should be removed.